plugins/modules/managed_serviceaccount.py

```diff
@@ -26,29 +26,51 @@
         type: str
         required: True
     wait:
-        description: Whether to wait for clusters to show up as managed clusters.
+        description: Whether to wait for managed-serviceaccount to show up.
         type: bool
         default: False
         required: False
     timeout:
-        description: Number of seconds to wait for the addons to show up.
+        description: Number of seconds to wait for the managed-serviceaccount to show up.
         type: int
         default: 60
         required: False
     managed_cluster:
         description: Name of managed cluster to create serviceaccount.
         type: str
         required: True
+    state:
+        description:
+        - Determines if managed-serviceaccount should be created, or deleted. When set to C(present), an object will be
+          created. If set to C(absent), an existing object will be deleted.
```

Comment:
Is C(...) some sort of formatting?

Reply:
When you run `ansible-doc -t module ocmplus.cm.managed_serviceaccount` , it will print out like this:

```
- state
        Determines if managed-serviceaccount should be created, or deleted. When set to `present', an object will be created. If set
        to `absent', an existing object will be deleted.
        (Choices: absent, present)[Default: present]
        type: str
```

Reply:
Coooooool!